cmd/mic/main.go

```diff
@@ -18,16 +18,27 @@ var (
 	forceNamespaced   bool
 	versionInfo       bool
 	syncRetryDuration time.Duration
+	leaderElectionCfg mic.LeaderElectionConfig
 )
 
 func main() {
 	defer glog.Flush()
+	hostName, err := os.Hostname()
+	if err != nil {
+		glog.Fatalf("Get hostname failure.")
```

Comment:
Need to include the error in the err msg

Reply:
Done.